exec/cnex/object.h

```diff
@@ -9,6 +9,7 @@
 #include "util.h"
 
 struct MmapObject;
+struct sqlite3;
```

Comment:
Why is this still here? You don't need ANY changes to object.h.

Reply:
Removed